plonetheme/barceloneta/theme/less/barceloneta-compiled.css

```diff
@@ -3922,6 +3942,25 @@ ul.picker__list {
 #content {
   margin-bottom: 48px;
 }
+#content-core a {
+  border-bottom: 1px solid #cccccc;
+}
+#content-core a:hover,
+#content-core a:focus {
+  border-bottom: none;
+}
+#content-core a:focus {
+  outline: thin dotted;
+  outline: 5px auto -webkit-focus-ring-color;
+  outline-offset: -2px;
+}
+#content-core .pat-structure a,
+#content-core .pat-resourceregistry a {
+  border-bottom: none;
```

Comment:
Looking good. Just one more thing(no underline in folder contents):

![screen shot 2015-10-25 at 10 18 23 am](https://cloud.githubusercontent.com/assets/524413/10716292/dcd0b1ec-7b01-11e5-9070-1aaa739d8556.png)

Also, on second thought, I'm a bit worried about the portal tabs active style being disruptive on a point release. I'm surprised that is an issue. The style is noticeably different. 

![screen shot 2015-10-25 at 10 18 53 am](https://cloud.githubusercontent.com/assets/524413/10716295/f88c2948-7b01-11e5-82e5-941c18503205.png)